examples/tools/tool-package-quickstart/my_tool_package/tools/utils.py

```diff
@@ -1,12 +1,14 @@
-import yaml
 from pathlib import Path
 
+from ruamel.yaml import YAML
+
 
 def collect_tools_from_directory(base_dir) -> dict:
     tools = {}
+    yaml = YAML()
     for f in Path(base_dir).glob("**/*.yaml"):
         with open(f, "r") as f:
-            tools_in_file = yaml.safe_load(f)
+            tools_in_file = yaml.load(f)
```

Comment:
please share the screenshot to show "my_tool_package" can list succesfully.

Reply:
![image](https://github.com/microsoft/promptflow/assets/111329184/1adcc06e-c262-412f-81e6-2130359d2a75)